File: TypeScriptTest/dictionary.js
//////////////////////////////////////////////////////////////////////

var Dictionary = (function () {
    "use strict";

    //////////////////////////////////////////////////////////////////////

    var dictionary = null;

    return {

        init: function (dict) {
            dictionary = dict;
        },

        words: function () {
            return dictionary.words;
        },

        isWord: function (str) {
            return dictionary.words.hasOwnProperty(str);
        },

        getDefinition: function (str) {
            return Dictionary.isWord(str) ? dictionary.words[str] : "??";
        }
    }

}());

File: TypeScriptTest/web_service/dictionary.py
import json

class Dictionary(object):
    
    def __init__(self):
        with open('dictionary.json') as json_data:
            self.dictionary = json.loads(json_data)
            json_data.close()

    def is_word(w):
        return w in self.dictionary

    def score(w):
        if is_word(w):
            pass
        else:
            return 0
